tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesAdapterWebConfiguration.java

```diff
@@ -1,10 +0,0 @@
-package com.epam.pipeline.tesadapter.configuration;
-
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
-
-@EnableWebMvc
-@Configuration
-public class TesAdapterWebConfiguration extends WebMvcConfigurerAdapter {
```

Comment:
does it need to be deleted?

Reply:
It is redundant to use auto-configuration provided by SpringBoot with class annotated with @EnableWebMvc in a same time.
If we use class with @EnableWebMvc we should configure all swagger-configuration in this class using for example for handling paths addResourceHandlers() method.